config.default.ini.php

```diff
@@ -11,6 +11,11 @@
 ; false = disabled (default)
 custom_timeout = false
 
+[misc]
```

Comment:
If you agree on changing the term from "maintainer" to "administrator", I suggest changing this line as well:
```suggestion
[admin]
```